### ISSUE TITLE
Lab03 Github merge

### ISSUE THREAD

**Issue by Leah-Coffin** [Watts-College/paf-515-template#8]
Hi @castower
I was following steps to merge my lab03 to github and encountered a conflict.  I'm not too confident if I should click on resolve conflict or if I made an error last week when merging that caused this week to not work. 

This branch has conflicts that must be resolved
Use the [web editor](https://github.com/Watts-College/project-paf-515-2024f-team-07/pull/19/conflicts) or the  to resolve conflicts.
Conflicting files
labs/wk03/imgs/README.md

**Comment by castower** [comment]
Hello @Leah-Coffin,

I was able to review the repo. 

A couple of things:
1. Your branch is currently 2 commits behind main so that is likely causing the issue.
2. The README.md doesn't match for the imgs folder. 

To resolve this, you can move your updated Lab 3 files to a separate folder (just to be sure there's no issues with the commit process). Then create a new branch, fetch to ensure it matches main, add your updated files (except the README.md file) back to the wk03 folder and then complete the pull request.

Let me know if you run into any further issues!

**Comment by Leah-Coffin** [comment]
@castower , thank you.  Do I do those steps in Github or on Github desktop?  I'm also not sure what you mean by moving my lab 3 files to a separate folder.  Is that in R, in github, or in my files that are on my computer?

**Comment by castower** [comment]
@Leah-Coffin 
Yes, you'll move the files on your computer and then repeat the steps to create a new branch in GitHub Desktop.
